WalletWasabi/Crypto/GroupElement.cs

```diff
@@ -59,5 +64,42 @@ public GroupElement(GEJ groupElement)
 		}
 
 		public static bool operator !=(GroupElement a, GroupElement b) => !(a == b);
+
+		/// <summary>
+		/// ToString is only used for nice visual representation during debugging. Do not rely on the result for anything else.
+		/// </summary>
+		public override string ToString()
+		{
+			if (IsInfinity)
+			{
+				return "Infinity";
+			}
+			else if (Ge.x == EC.G.x && Ge.y == EC.G.y)
```

Comment:
This is simply another point and should not have a different status.

Reply:
It's the standard generator point, it's not any point.